### DISCUSSION TITLE
v4中的数据清洗，高精度模型的训练数据和要清洗的数据相同么？

### DISCUSSION THREAD

**Discussion by Zhaohuii-Wang** [PaddlePaddle/PaddleOCR#12325] [Q&A]
v4中的数据清洗，高精度模型的训练数据和要清洗的数据相同么？如果不相同的话，数据分别都有哪些呢？

**Comment by cuicheng01** [comment]
麻烦您再详细描述一下您的问题呢？

**Comment by Zhaohuii-Wang** [comment]
您好，感谢回复，我再详细描述一下我的问题，并在此基础上提出几个新的疑问，希望得到解答。就是ppocrv4中识别模型的第一个优化方法DF中提到的高精度模型和低精度模型，其中高精度模型说是使用了ppocrv3，我想问的就是，训练ppocrv3使用的训练数据和df中需要清洗的训练数据是同一批么？如果不同的话，有什么区别呢？低精度模型说是使用少量数据训练，这个比例和数据的选择有什么讲究，如果直接使用所有数据仅训练一个epoch能不能达到类似的效果呢，有没有做过相应的实验分析？
